Xcode/LiteCore.xcodeproj/project.pbxproj

```diff
@@ -432,10 +432,10 @@
 			fileType = sourcecode.java;
 			isEditable = 1;
 			outputFiles = (
-				"$(DERIVED_FILE_DIR)/Java_com_couchbase_litecore_Database.h",
+				"$(DERIVED_FILE_DIR)/Java_com_couchbase_lite_internal_Database.h",
 				"",
 			);
-			script = "set -e\n/usr/bin/javah -classpath $SRCROOT/../java/src -d $DERIVED_FILE_DIR com.couchbase.litecore.$INPUT_FILE_BASE";
+			script = "set -e\n/usr/bin/javah -classpath $SRCROOT/../java/src -d $DERIVED_FILE_DIR com.couchbase.lite_internal.$INPUT_FILE_BASE";
```

Comment:
Shall we do `com.couchbase.lite.internal.{native | jni}` instead of `lite_internal`?